app/models/manageiq/providers/openstack/inventory_collection_default/cloud_manager.rb

```diff
@@ -0,0 +1,269 @@
+class ManageIQ::Providers::Openstack::InventoryCollectionDefault::CloudManager < ManagerRefresh::InventoryCollectionDefault::CloudManager
+  class << self
+    def availability_zones(extra_attributes = {})
+      attributes = {
+        :inventory_object_attributes => [
```

Comment:
we should also have here a correct STI model class, then we can remove the type fro ma parser (except the Provate Public network case)